src/components/footer/footer.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./../../styles.scss";
+import Code4RomaniaGrayLogo from "../../../images/code4romania-gray.svg";
+import "./footer.scss";
+
+const Footer = ({ children }) => {
+  return (
+    <footer className="App-footer">
+      <div className="footer-content">
+        {children}
+        <div className="code4-branding">
+          <a
+            href="https://code4.ro/ro/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {/* <img src={Code4romania_gray} alt="Code for Romania gray logo"></img> */}
+            <Code4RomaniaGrayLogo style={{ height: 40 }} />
+          </a>
+          <p>© 2020 Code for Romania.</p>

```

Comment:
It would be a good idea to make the current year dynamic here

Reply:
Good idea! Done.